### PR TITLE
Created mobile version of node page

### PR DESCRIPTION
**Pull request by alexandercyu**
Applied changes from mobile design

### CODE REVIEW

File: templates/site/node.html [hunk 1]
```diff
@@ -14,20 +16,24 @@
     {% include "partials/banner.html" %}
 {% endwith %}
 
-{% with %}
-    {% set title = "Start a Project" %}
-    {% set subtitle = "Select most suitable node or server to start a project" %}
-
-    {% include "partials/general-title-subtitle.html" %}
-{% endwith %}
+    <div class="container-fluid d-none d-lg-block mt-5 w-100">
```
Comment (mishaschwartz) [inline templates/site/node.html:19]: This only shows on lg screen? I see that's what it shows in Figma but are we sure? @ShrutiK100

File: static/css/marble.css [hunk 2]
```diff
@@ -946,4 +872,228 @@ h6{
...
+    /*Node Page*/
+    .value-statement-node{
+        width: 85%;
+        text-align: center;
+      }
+
+    .node-menu-header{
+        font-family: var(--secondary-font-family);
+        font-weight: var(--reg-font-weight);
+        font-size: 20px;
```
Comment (mishaschwartz) [inline static/css/marble.css:890]: This may look fine on a `sm` size but on `md` it looks really small

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
Looks good!

The only other issue is that the `#nodeServices` div has its content aligned with the start of the flexbox which means that the services are shifted to the left of the screen.

**Comment by alexandercyu** [comment]
> Looks good!
> 
> The only other issue is that the `#nodeServices` div has its content aligned with the start of the flexbox which means that the services are shifted to the left of the screen.

This is so when the services cards don't take up the full width of the row, the remaining one doesn't centre align and instead, starts at the beginning of the row.

File: static/css/marble.css (hunk 2 above)
Comment (alexandercyu) [reply static/css/marble.css:890]: @ShrutiK100 , can you advise on the font size to use?

**Comment by mishaschwartz** [comment]
> This is so when the services cards don't take up the full width of the row, the remaining one doesn't centre align and instead, starts at the beginning of the row.

I don't see that as a problem. Even if it is, the items should still not be awkwardly shifted to the left of the page.

**Comment by alexandercyu** [comment]
> > This is so when the services cards don't take up the full width of the row, the remaining one doesn't centre align and instead, starts at the beginning of the row.
> 
> I don't see that as a problem. Even if it is, the items should still not be awkwardly shifted to the left of the page.

Also, on some screen sizes (1920x1080) the row hasn't expanded enough to allow the service card to go to more than 3 per line.  That would make it look like it's shifted to the left of the screen.

**Review by ShrutiK100 — CHANGES_REQUESTED** [review]
As discussed in the meeting, all services should be center aligned.

File: static/css/marble.css (hunk 2 above)
Comment (alexandercyu) [reply static/css/marble.css:886]: Made the font size h3 size in screens larger than phone

**Comment by alexandercyu** [comment]
> > This is so when the services cards don't take up the full width of the row, the remaining one doesn't centre align and instead, starts at the beginning of the row.
> 
> I don't see that as a problem. Even if it is, the items should still not be awkwardly shifted to the left of the page.

Fixed the spacing and alignment issue.

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
Ok a few things: 

- on `sm` size the services are still not centre aligned
- there's a screen width around the transition between `sm` and `md` where it goes from two columns to one column and then back to two columns. I think this is caused by the changing width of the flex items.

**BUT** I don't think it's worth spending any more time on this. Please just clean up the code (see https://github.com/DACCS-Climate/marble-website/pull/126#discussion_r1602405251) and then this is good enough.

**Comment by alexandercyu** [comment]
>  * on sm size the services are still not centre aligned

Do you mean when the browser is shrunk to the smallest width?  It could be caused by the card text being left aligned and "adding" space on the right.

Screenshot 2024-05-16 at 10 59 26 AM (2)Crop

If not, then it's likely caused by the bootstrap class justify-content-between while the screen size is transitioning.


>     * there's a screen width around the transition between `sm` and `md` where it goes from two columns to one column and then back to two columns. I think this is caused by the changing width of the flex items.

That's what it is.  At desktop size the width of the parent div is 75% so the service cards are centred and the entire section is aligned with the rest of the page.  At lower screen sizes the parent div increases to 90% for the same reason.

I tried a quick solution by adjusting the width slightly larger at each breakpoint but it still keeps happening.

File: static/css/marble.css [hunk 3]
```diff
@@ -973,6 +969,126 @@ h6{
...
+
+    .services-button-margin{
+        margin-top: 10px;
+    }
+}
+
+@media (width >= 576px) {
+
+    /*Node Page*/
+    .value-statement-node{
```
[the remark this answers is not in the document]
Comment (alexandercyu) [reply static/css/marble.css]: Rearranged the css into their proper media queries.

**Review by mishaschwartz — APPROVED** [review]

File: templates/site/node.html (hunk 1 above)
Comment (ShrutiK100) [reply templates/site/node.html:19]: I just realized while reviewing that I haven't added the heading "Start the project" and body under it. I have added it in the Figma design now

File: static/css/marble.css (hunk 2 above)
Comment (ShrutiK100) [reply static/css/marble.css:1020]: This is which heading exactly?

**Review by ShrutiK100 — CHANGES_REQUESTED** [review]
The node page looks good! Just a couple of changes, I missed adding the title "Start your project" above the node nav bar, please add that Heading  leaving 50px above the heading 30px below. Add 50px space between node banner Title and subtitle.

**Comment by alexandercyu** [comment]
> The node page looks good! Just a couple of changes, I missed adding the title "Start your project" above the node nav bar, please add that Heading leaving 50px above the heading 30px below. Add 50px space between node banner Title and subtitle.

Added the "Start Your Project" intro section back in.  

Note that the 50px space between this intro section and the node buttons (between content sections) was added in the marble-mobile-home branch.  The space will be applied when this branch and the home branch is merged in .

**Comment by alexandercyu** [comment]
> Remove the space dividing the Title Screenshot 2024-05-22 at 2 50 50 PM
> 
> Fix node nav bar alignment issue occurring in Tablet size browser Screenshot 2024-05-22 at 3 04 57 PM
> 
> Add 30px gap since we have only 3 nodes, once we have 4 nodes we can reduce it back to 20 px Screenshot 2024-05-22 at 2 59 05 PM

- The space in the title could have been caused by a conflict.  It should already be gone.
- Space between menu items in tablet size fixed
- When the number of menu items increases higher than 3 it will be put into the "Other" dropdown.  No need to change spacing between menu items every time a node is added

**Comment by alexandercyu** [comment]
NOTE: I changed the font size of the email to 12px because the 16px font was making it wrap to two lines

**Review by mishaschwartz — APPROVED** [review]
Looks good. I say let's merge this and then we can adjust as needed after our final review

**Pull request merged**